content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,200 @@
+---
+title: June '22 Community Gems
+date: 2022-06-29
+description: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+picture: 2022-06-29/june-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/may-22-community-gems/1184
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - GCP
+  - Community
+---
+
+## [Is there something like `git add-commit -m "some message"` for DVC?](https://discord.com/channels/485586884165107732/563406153334128681/981498675689828362)
```

Comment:
Had to look up the original question; it sounds like Ramnath wants to do all of this with one command. I'm not sure the explanation now answers that question. Maybe go for "No, but you could create an alias"?